pandas/indexes/base.py

```diff
@@ -352,6 +339,42 @@ def __new__(cls, data=None, dtype=None, copy=False, name=None,
     """
 
     @classmethod
+    def _convert_to_int_index(cls, data, copy, name):
+        """
```

Comment:
is there any case where the dtype is specified when you are calling this? (IOW should there be)